apps/jobbrowser/src/jobbrowser/api2.py

```diff
@@ -119,7 +119,7 @@ def action(request, interface=None, action=None):
 
   response['operation'] = operation
   response.update(
-      get_api(request.user, interface, cluster=cluster).action(app_ids, operation)
+      get_api(request.user, interface, cluster=cluster).action(app_ids, operation, request)
```

Comment:
Isn't this going to break all the interface implementing action()? Could we avoid adding the parameter?

Reply:
It was added in the following commit
https://github.com/cloudera/hue/commit/e2f186b66495650e25e6503a3575ec592e7bb669#diff-01c5d69d9fddda43741beccd06725c538e3e525ab2d0c78478d216b015986524R122

But was removed sometime later.... thought that was removed by mistake, hence re-added in this commit.

It's required for the Kill action to not return the request until kill is made.
Right now kill is broken.

Moving the change to a separate patch, for further discussion.

Reply:
Submitted a fresh patch @ https://github.com/cloudera/hue/pull/1597